scripts/eslint-pre-commit

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
-
-git stash -q --keep-index
-
-# Test prospective commit
-git diff-index --cached HEAD --name-only --diff-filter ACMR | egrep '.js$' | xargs -I $(npm bin)/eslint {}
```

Comment:
Instead of deleting the whole file, and running lint on even unstaged changes, you can just lint the files with changes

```
git diff-index --cached HEAD --name-only --diff-filter ACMR | egrep '.js$' | xargs -I $(npm bin)/eslint {}
```

If you're fine with getting errors for files you're still working on, then it's no biggie